### ISSUE TITLE
Prepare devcontainers repository for public

### ISSUE THREAD

**Issue by yevhenkuznetsov** [yevhenkuznetsov/devcontainers#7]
In order to pull devcontainers without permissions, the following action items has to be fulfiled:
- [ ] Document and add examples how images can be used to start a development container

As a nice to have feature:
- [ ] Add an additional jobs for PRs to build images before merging. Re-use images after PR merge to speed-up the deployment.

Depends on #10
